docs/4_4_1_can.adoc

```diff
@@ -662,16 +689,16 @@ As soon as an FMU loses arbitration in this way, it shall independently repeat t
 [#figure-can-arbitration]
 image::can_arbitration.svg[width=70%, align="center"]
 
-Within a <<ConfigurationCanOpCode, Configuration operation>>, the `Buffer` argument can be specified.
-Once this is set, the Bus Simulation buffers the frame after losing arbitration and sends it as soon as possible.
-When using this parameter, it is therefore not necessary for the FMU to trigger the respective frame to be sent again.
-In this scenario the <<ArbitrationLostOpCodeCanLowCut, Arbitration Lost operation>> shall not be returned to the specific FMU.
+Within a <<ConfigurationCanOpCode, Configuration operation>>, the `Arbitration Lost Behavior` argument can be specified.
+Once this option is deactivated, the Bus Simulation buffers the frame after losing arbitration and sends it as soon as possible.
+In this case it is not necessary for the Network FMU to trigger the respective frame to be sent again and an <<ArbitrationLostOpCodeCanLowCut, Arbitration Lost operation>> shall not be returned to the specific Network FMU.
+If the `Arbitration Lost Behavior` is activated, the specified Network FMU is informed by an <<ArbitrationLostOpCodeCanLowCut, Arbitration Lost operation>> and needs to repeat the corresponding <<TransmitOpCodeCanLowCut, Transmit operation>> itself.
```

Comment:
'activated' and 'deactivated': argument true/false? We should be clear about how de-/activation is achieved.

Reply:
We already define it in line 470: The parameter value is defined as `Deactivated = 0` and `Activated = 1`. But I think we can repeat the information in the marked position to make it more clear for a reader. Is this OK for you @andreas-junghanns? If it is I´m going to fix it.